phoenicis-repository/src/test/java/org/phoenicis/repository/repositoryTypes/CachedRepositoryTest.java

```diff
@@ -38,8 +38,8 @@ public void testFetchInstallableApplications() throws Exception {
                         .thenReturn(
                                 new RepositoryDTO.Builder()
                                         .withCategories(
-                                                Arrays.asList(new CategoryDTO.Builder().withName("Category 1").build(),
-                                                        new CategoryDTO.Builder().withName("Category 2").build()))
```

Comment:
So we could have two kinds of DTOs (translated and untranslated) at the end.
I suggest we have a Translatable interface then